.circleci/test-deploy.yml

```diff
@@ -13,21 +13,45 @@ jobs:
       - cypress/install:
           working-directory: examples/angular-app
           cypress-cache-key: cypress-cache-{{ arch }}-{{ checksum "examples/angular-app/package.json" }}
+          include-branch-in-node-cache-key: true
       - persist_to_workspace:
           root: ~/
           paths:
             - .cache/Cypress
             - project
-  run-tests-in-parallel:
-    executor: cypress/default
-    parallelism: 3
+  run-tests-in-chrome:
+    docker:
+      - image: cypress/browsers:node-16.18.1-chrome-109.0.5414.74-1-ff-109.0-edge-109.0.1518.52-1
```

Comment:
Where did we land on setting these images? Do these need to be this specific?

Reply:
Yeah mirrors the current functionality per @jennifer-shehane